### PR TITLE
10 reflection

### PR DESCRIPTION
**Pull request by SlavaPeshko**

### CODE REVIEW

**Comment by AlexMokin** [comment]
well done!

File: 10-Reflection/Reflection.Tasks/CommonTasks.cs [hunk 1]
```diff
@@ -60,10 +73,28 @@ public static class CommonTasks
...
-        }
+			// TODO : Implement SetPropertyValue method
+			//throw new NotImplementedException();
+
+			var ass = obj.GetType();
+			var propertyPaths = propertyPath.Split('.');
+
+			if (propertyPaths.Length == 1)
+			{
+				obj.GetType().BaseType.GetProperty(propertyPath).SetValue(obj, value);
```
Comment (AlexMokin) [inline 10-Reflection/Reflection.Tasks/CommonTasks.cs]: what if you need to go deeper https://allisonmaruska.files.wordpress.com/2015/10/we-need-to-go-deeper.jpg

**Comment by SlavaPeshko** [comment]
update method SetPropertyValue

**Comment by AlexMokin** [comment]
Good job!

**Pull request closed without merging**